packages/cli/src/imports.js

```diff
@@ -5,24 +5,22 @@ const localNodeModules = path.resolve(process.cwd(), 'node_modules')
 
 // Prefer importing modules from local node_modules (for NPX and global bin)
 async function _import(modulePath) {
-  let m
-  for (const mp of [ path.resolve(localNodeModules, modulePath), modulePath ]) {
+  for (const mp of [
+    path.resolve(localNodeModules, modulePath),
+    modulePath
+  ]) {
     try {
```

Comment:
Is await for more readable?

Reply:
Without await, (and returning promise) we cannot catch error here. One way would be separating it like this:

```js
const m = await import(mp)
return m
} catch(e) [
```

But I taught one-liner would be more readable.

Reply:
Will throw inside catch instead of end be better?

Reply:
We have to only throw an error if all possible locations (for loop) has been checked. Throwing inside the catch of for loop requires one additional check (`m === mp`) to see if we are reaching to the end or not. This is same as refactoring out of for loop and using early return

Reply:
Gotcha